src/main/java/stsjorbsmod/actions/ExposeAction.java

```diff
@@ -31,11 +33,18 @@ public void update() {
         tickDuration();
         if (isDone) {
             this.target.damage(this.info);
-            if (AbstractDungeon.getCurrRoom().monsters.areMonstersBasicallyDead()) {
-                AbstractDungeon.actionManager.clearPostCombatActions();
-            } else {
+
+            boolean nonMinionsLeft = false;
+            for (AbstractMonster m : AbstractDungeon.getCurrRoom().monsters.monsters) {
+                if (!m.isDeadOrEscaped() && !m.hasPower(MinionPower.POWER_ID)) {
+                    nonMinionsLeft = true;
+                }
+            }
+            if (nonMinionsLeft) {
                 AbstractPlayer p = AbstractDungeon.player;
                 addToBot(new LoseHPAction(p, p, hpLoss));
+            } else {
+                AbstractDungeon.actionManager.clearPostCombatActions();
```

Comment:
clearPostCombatActions should probably still be under a separate `areMonstersBasicallyDead` check per usual, rather than in an else clause here